.github/workflows/gradle.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build
+
+on: [pull_request, push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
Why Java 1.8. You can find rationale for using JDK 11 [here](https://github.com/Spottedleaf/Starlight/pull/35#discussion_r590564434).

Reply:
The vanilla launcher continues to ship java 8 by default, and although it's almost certain there will be an update to 15 in the coming weeks, mods should continue targeting the version of java the MC runs

Reply:
> The vanilla launcher continues to ship java 8 by default, and although it's almost certain there will be an update to 15 in the coming weeks, mods should continue targeting the version of java the MC runs

Here is an explanation for that point: https://github.com/Spottedleaf/Starlight/pull/35#discussion_r590634109